### ISSUE TITLE
Another file name convention question

### ISSUE THREAD

**Issue by hafen** [hendrycks/natural-adv-examples#3]
Adding to #2, just to clarify, with the following path:

`n01498041/0.000348_chameleon _ box turtle_0.55540705.jpg`

the true class for the image is "stingray" (from the directory `n01498041`), but it was predicted as a "box turtle" with confidence 0.55540705. So how do I interpret the `0.000348_chameleon` part of the file name?

**Comment by hendrycks** [comment]
There were two ResNets, one fine-tuned on the 200 classes and one which was not. The one which was not fined tuned had its 800 logits masked. This is why there are two predictions.

**Comment by expectopatronum** [comment]
Is `0.000348` the confidence for `chameleon`? How can this be the top predicted class?

**Comment by hendrycks** [comment]
The first probability, if I recall correctly, is the probability assigned to the correct class.
